pkg/commands/validate/get_command.go

```diff
@@ -23,15 +23,14 @@ import (
 )
 
 // AddCommandsTo adds commands to a root cobra command.
-func AddCommandsTo(ctx context.Context, fio files.FileReaderWriter, sio cmdlib.StdioReaderWriter, root *cobra.Command) {
+func GetCommand(ctx context.Context, fio files.FileReaderWriter, sio cmdlib.StdioReaderWriter, gopts *cmdlib.GlobalOptions) *cobra.Command{
```

Comment:
Why this change?

Reply:
Make sure to run go lint, go vet -- looks like a bunch of method comments didn't get changed.

Reply:
Yea I need to remove them.

Reply:
So in detail if you want to know why:
opts was package level despite usage at function level.  This is effectively wrong.  Multiple calls to the function result in multiple references to the same static data.  same applies to gopts. 
get vs add since I was in the neighborhood.  Nice to restrict side effect to rootcmd at a single scope, just makes things easier to reason about.  :+1: